temporalcli/commands.workflow_exec_test.go

```diff
@@ -0,0 +1,346 @@
+package temporalcli_test
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"net/http/httptest"
+	"os"
+	"strconv"
+
+	"github.com/google/uuid"
+	"go.temporal.io/api/common/v1"
+	"go.temporal.io/api/enums/v1"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/sdk/converter"
+	"go.temporal.io/sdk/worker"
+	"go.temporal.io/sdk/workflow"
+	"google.golang.org/protobuf/proto"
+)
+
+// TODO(cretz): To test:
+// * Start failure
+// * Execute failure on start
+// * Execute workflow failure (including nested failures)
+// * Execute workflow cancel
+// * Execute workflow timeout
+// * Execute workflow continue as new
+// * Workflow with proto JSON input
+
+func (s *SharedServerSuite) TestWorkflow_Start_SimpleSuccess() {
+	// Text
+	s.Worker.OnDevWorkflow(func(ctx workflow.Context, input any) (any, error) {
+		return map[string]string{"foo": "bar"}, nil
+	})
+	res := s.Execute(
+		"workflow", "start",
+		"--address", s.Address(),
+		"--task-queue", s.Worker.Options.TaskQueue,
+		"--type", "DevWorkflow",
+		"--workflow-id", "my-id1",
+	)
+	s.NoError(res.Err)
+	// Confirm text has key/vals as expected
+	out := res.Stdout.String()
+	s.ContainsOnSameLine(out, "WorkflowId", "my-id1")
+	s.Contains(out, "RunId")
```

Comment:
I think our uses of `Contains` and `ContainsOnSameLine` are all doing the same thing: they're asserting that there exists a line in which a sequence of strings occurs. The "sequence" might just be one string, and the strings may be separated by unspecified characters. So I think we should use the same assertion utility for sequences of length 1 and length > 1.